index.d.ts

```diff
@@ -238,11 +238,10 @@ declare namespace Rollbar {
     enabled?: boolean;
     autoStart?: boolean;
     maxSeconds?: number;
-    triggerOptions?: {
-      item?: {
-        levels?: Level[];
-      };
-    };
+    triggers?: {
+      type: string;
+      level?: string[];
```

Comment:
We have a type for this `Level[]` that's defined as:
```ts
export type Level = 'debug' | 'info' | 'warning' | 'error' | 'critical';
```

Reply:
Updated